hw/ip/aes/doc/dv/index.md

```diff
@@ -49,6 +49,10 @@ AES instantiates (already handled in CIP base env) [tl_agent]({{< relref "hw/dv/
 which provides the ability to drive and independently monitor random traffic via
 TL host interface into AES device.
 
+### EDN agent
+
+AES instatiates (already handles in the CIP base env) [edn_agent]({{< relref "hw/ip/edn/doc/dv/index.md" >}})
```

Comment:
Nit: insta**n**tiates.

Reply:
I don't think that is a nit :D - but fixed